util/util.py

```diff
@@ -167,7 +167,8 @@ def remove_challenge_by_channel_id(database, challenge_channel_id, ctf_channel_i
     """
     ctfs = pickle.load(open(database, "rb"))
     ctf = ctfs[ctf_channel_id]
-    ctf.challenges = list(filter(lambda challenge: challenge.channel_id != challenge_channel_id, ctf.challenges))
+    ctf.challenges = list(filter(
```

Comment:
Same as: `[challenge for challenge in ctf.challenges if challenge.channel_id != challenge_channel_id]`.

You can keep as is, I find the list comprehensions to be easier to read.

Reply:
This is just a reformat, so let's keep it for now.